apps/bettafish/src/app/pages/browse/browse.component.html

```diff
@@ -6,7 +6,7 @@ <h3>Browse</h3>
                 <rmx-icon name="compass-3-line"></rmx-icon>
             </div>
             <div class="pages">
-                <a class="link" [routerLink]="['/browse']" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }">
+                <a class="link" (click)="setTitle()" [routerLink]="['/browse']" [routerLinkActive]="'active'" [routerLinkActiveOptions]="{ exact: true }">
```

Comment:
When going from ex: /browse/newest-works to /browse, ngOnInit() isn't triggered, So, when clicking the link, it manually sets the title. This doesn't handle backing into those pages, but that can be handled later.